pkg/manifests/bundle.go

```diff
@@ -21,6 +21,8 @@ type Bundle struct {
 	Dependencies   []*Dependency
 	// CompressedSize stores the gzip size of the bundle
 	CompressedSize *int64
+	// Size stores the size of the bundle
```

Comment:
nit: the indenting looks messed up for this struct chunk. Does running `go fmt` on it fix it maybe?

Reply:
I think it is solved. The ci checks the go fmt.